lib/main/audio/AudioStreamManager.ts

```diff
@@ -0,0 +1,113 @@
+import { AudioChunkSchema } from '@/app/generated/ito_pb'
+import { create } from '@bufbuild/protobuf'
+
+export class AudioStreamManager {
```

Comment:
@JohnDonavon would appreciate your eyes on this especially. I rebased to include your changes for the 100ms